changelog/unreleased/issue-1681

```diff
@@ -0,0 +1,11 @@
+Bugfix: mount: change behavior of `restic mount` when path doesn't exist
+
+We've change the behavior of `restic mount` command when the directory doesn't
+exist, prevously restic was trying to create the mount point path in that case,
+this is not the current behavior anymore, we now return an explicit error.
+In addition to this, we check that we have permission to write in the directory.
+
+
```

Comment:
Thanks for adding the changelog! Please update it to the following:

~~~
Bugfix: Make mount check permissions and not create missing mount point

When specifying a non-existent directory as mount point for the mount
command, restic used to create the directory automatically. This has now
changed such that restic instead gives an error when the directory for the
mount point doesn't exist.

Restic now also checks the permissions of the specified mount point to
make sure that they are sufficient for mounting the repository in it.
~~~

Also please remove the excessive empty line before the URLs at the end :)

Reply:
Thanks, done!